app/src/main/java/com/github/fribourgsdp/radio/game/prep/JoinGameActivity.kt

```diff
@@ -106,4 +115,36 @@ open class JoinGameActivity : MyAppCompatActivity() {
             joinErrorView.visibility = View.VISIBLE
         }
     }
+
+    private fun initPublicLobbiesDisplay() {
+        // init the cards
+        lobbiesRecyclerView.layoutManager = LinearLayoutManager(this)
+        val adapter = PublicLobbiesAdapter(this, this.db).apply {
```

Comment:
Here could you not replace this.db with db?

Reply:
Done in pr #242